cypress/page/products-list.page.ts

```diff
@@ -1,14 +1,19 @@
 class ProductListPage {
     private additemBtn: string;
     private checkoutItem: string;
+    private allProducts: string;
   
     constructor() {
       this.additemBtn = ".button-container a[title='Add to cart']";
       this.checkoutItem = ".button-container a[title='Proceed to checkout']";
+      this.allProducts = ".product-container";
     }
   
-    public addToCart(): void {
-      cy.get(this.additemBtn).click();
+    public addTShirtToCart(product:string): void {
+      this.findProductByName(product).find(this.additemBtn).click();
+    }
+    private findProductByName (product:string) {
```

Comment:
add a blank line to separate the addTShirtToCart from the findProductByName  method

Reply:
all done @holgiosalos